src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private String color;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return this.color;
+    }
+}
```

Comment:
According to the checklist, don't use abstract classes to set behavior for classes. Consider whether `Figure` should be an abstract class or if you should use an interface to define the common behavior for all figures. If `Figure` only provides common properties without defining any abstract methods that subclasses must implement, you might want to use a regular class instead.